charts/global-network-policies/templates/allow-to-metadata.yaml

```diff
@@ -11,11 +11,13 @@ spec:
   podSelector:
     matchLabels:
       networking.gardener.cloud/to-metadata: allowed
-{{- if .Values.metadataService }}
+{{- if .Values.blockedAddresses }}
   egress:
+{{- range $i, $address := .Values.blockedAddresses }}
   - to:
     - ipBlock:
-        cidr: {{ .Values.metadataService }}
+        cidr: {{ $address }}
```

Comment:
The problem with this change is that it doesn't do what the `NetworkPolicy`'s name and description says it does.